recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
 
 test:
     requires:
-        - python {{ environ['PY_VER'] + '*' }}  # [win]
+        - python {{ PY_VER ~ '*' }}  # [win]
```

Comment:
BTW we might not need any change at all:

https://github.com/conda/conda-build/pull/957